modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/instance/SharedRocksDbInstance.java

```diff
@@ -395,7 +407,10 @@ private static byte[] metaPrefix(byte[] metaPrefix, byte[] tableIdBytes) {
     }
 
     private ColumnFamily createSortedIndexCf(byte[] cfName) {
-        ColumnFamilyDescriptor cfDescriptor = new ColumnFamilyDescriptor(cfName, sortedIndexCfOptions(cfName));
+        // TODO: CfOptions not closed!!!!
```

Comment:
The TODO should either be resolved or mention a Jira issue

Reply:
Sorry forgot to remove the todo